packages/template-retail-react-app/app/theme/foundations/shadows.js

```diff
@@ -5,5 +5,6 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 export default {
-    top: '0px -1px 3px rgba(0, 0, 0, 0.1), 0px -1px 2px rgba(0, 0, 0, 0.06)'
+    top: '0px -1px 3px rgba(0, 0, 0, 0.1), 0px -1px 2px rgba(0, 0, 0, 0.06)',
+    outline: '0 0 0 3px var(--chakra-colors-blue-500)'
```

Comment:
Blue 500 is the minimum value of blue colors that satisfies the constrast ratio against #ffffff background
![image](https://github.com/SalesforceCommerceCloud/pwa-kit/assets/52219283/4083efca-8ce1-4497-b98f-ad389f1fd998)